niworkflows/interfaces/utils.py

```diff
@@ -616,7 +614,7 @@ class AddTSVHeader(SimpleInterface):
 
     .. testcleanup::
 
-    >>> tmpdir.cleanup()
+    >>> os.chdir(cwd)
```

Comment:
We clean up in the fixture, so just getting out. The `chdir`s into directories that get deleted are ultimately what's causing the `pytest-cov` weirdness.